test/components/previews/pathogen/button_preview/trailing_visual.html.erb

```diff
@@ -1,4 +1,4 @@
 <%= render Pathogen::Button.new(scheme:, size:) do |button| %>
-  <% button.with_trailing_visual_icon(icon: ICON::ARROW_RIGHT) %>
+  <% button.with_trailing_visual_icon(icon: "arrow-right") %>
```

Comment:
suggestion: Shouldn't this be a symbol? or are strings also allowed?

Reply:
Updated in 9e4af5794cef49d802b294f603b8e6248e73b64b, both strings and symbols